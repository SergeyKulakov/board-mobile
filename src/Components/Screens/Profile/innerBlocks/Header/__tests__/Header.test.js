import React from 'react'
import { shallow } from 'enzyme'

import 'jest-styled-components'

import Component from '../Header'

describe('Profile.Header', () => {
  const defaultProps = {
    onGoBack: () => null,
    onClickLanguageButton: () => null,
    t: () => null,
    getCurrentLanguage: () => 'en',
  }

  it('render snapshot', () => {
    const wrapper = shallow(<Component {...defaultProps} />)

    expect(wrapper).toMatchSnapshot()
  })
})
