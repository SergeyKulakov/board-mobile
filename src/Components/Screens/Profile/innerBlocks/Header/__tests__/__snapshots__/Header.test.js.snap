// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile.Header render snapshot 1`] = `
<Component>
  <Styled(View)>
    <Styled(View)>
      <Icon
        color="#fff"
        name="angle-left"
        onClick={[Function]}
        size={40}
      />
    </Styled(View)>
    <Styled(View)
      flex={5}
    >
      <ScreenTitle
        style={
          Object {
            "fontSize": 25,
            "paddingTop": 3,
          }
        }
      />
    </Styled(View)>
    <Styled(View)>
      <Styled(View)>
        <Component
          flagName="en"
          onClick={[Function]}
        />
      </Styled(View)>
    </Styled(View)>
  </Styled(View)>
</Component>
`;
